dss-api.yml

```diff
@@ -252,7 +252,121 @@ paths:
                 format: date-time
             required:
               - version
-
+  /subscriptions/{uuid}:
+    get:
+      summary: Retrieve an event subscription given a UUID.
+      description: |
+        Given a subscription UUID, return the latest version of that bundle.  If the version is provided, that version of the
+        bundle is returned instead.
+      parameters:
+        - name: uuid
+          in: path
+          description: Subscription unique ID.
+          required: true
+          type: string
+          pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+        - name: replica
+          in: query
+          description: Replica to fetch from. Can be one of aws, gcp, or azure.
+          required: true
+          type: string
+          enum: [aws, gcp, azure]
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+            properties:
+              subscription:
+                $ref: "#/definitions/Subscription"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      summary: Create a event subscription.
+      description: |
+        Create a new event subscription. Every time an added bundle matches this query, 
+        a request is sent to callback_url.
+      parameters:
+        - name: uuid
+          in: path
+          description: A RFC4122-compliant ID for the subscription.
+          required: true
+          type: string
+          pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+        - name: replica
+          in: query
+          description: Replica to write to. Can be one of aws, gcp, or azure.
+          required: true
+          type: string
+          enum: [aws, gcp, azure]
+        - name: extras
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              query:
+                description: Elasticsearch query that will trigger the callback.
+                type: object
+              creator_uid:
+                description: User ID who is creating this subscription.
+                type: integer
+                format: int64
+              callback_url:
+                description: Url to send request to when a bundle comes in that matches this query.
+                type: string
+            required:
+              - query
+              - creator_uid
+              - callback_url
+      responses:
+        201:
+          description: OK
+          schema:
+            type: object
+            properties:
+              timeAdded:
```

Comment:
Not necessarily, but I thought it would be nice to return some JSON response because the cli automatically parses and prints the json response. Can change that behavior, but I think it's nice to get some output. 

wrt the contents of the response, I didn't think there was much relevant information to return so the most obvious thing was a similar response to what is returned after PUT bundles/files.